api/sheets/get_name.md

```diff
@@ -6,41 +6,62 @@ parent: Google Sheets
 
-### Example 2: Retrieve the names of a collection of cell lines.
+![alt_text](images/sheets_get_return_error_1.png "image_tooltip")
 
-![](/assets/images/sheets/sheets_get_name_cell_lines_cropped.png)
+If you provide an empty cell for a DCID, the `=DCGETNAME(*dcids*)` formula returns a value of #ERROR!, as shown show in the following image:
 
->  **TIP:**
->
->  It's best to minimize the number of function calls to `DCGETNAME` by using a single call to get names for a column of nodes. This is because a spreadsheet will make one call to a Google server [per custom function call](https://developers.google.com/apps-script/guides/sheets/functions#optimization). If your sheet contains thousands of separate calls to `DCGETNAME`, expect it to be slow and with errors.
+![alt_text](images/sheets_get_return_error_2.png "image_tooltip")
 
+## Examples {#examples}
 
-## Error returns
+This section contains examples of returning the names associated with given node DCIDs.
 
-If a DCID does not exist, this method returns nothing:
+### Example 1: Retrieve the Name of a Country by its DCID
 
-![](/assets/images/sheets/sheets_get_name_wrong_dcid_cropped.png)
+To retrieve the name of a country by its DCID, perform the following steps:
 
-If an empty cell is provided as a DCID, this method returns an error value:
+1. Place your cursor in the cell where you want to add a DCID. In this case, cell A1.
+2. Click in the search box for the **Fill place dcids for selected cells** section. For this example, enter “Ivory Coast”. The cell is populated with a DCID of “country/CIV”.
+3. Move to the cell where you want to retrieve the country name.
+4. Enter the formula **`=DCGETNAME(A1)`** to retrieve the country name and press **Enter**. The French spelling of Ivory Coast (Côte d'Ivoire) populates the cell.
+
+![alt_text](images/sheets_get_name_example_1.png "image_tooltip")
+
+### Example 2: Retrieve the Names of a Collection of Cells
+
+To retrieve the names of a collection of cells using DCIDs, perform the following steps:
+
+1. Using the following image, enter the DCIDs for four African nations into column A.
+2. Place your cursor in cell B1 and type the following formula: **`=DCGETNAME`**.
+3. Press **Enter** and the names of the four African nations populates column B.
+
+![alt_text](images/sheets_get_name_example_2.png "image_tooltip")
+
+> **IMPORTANT**
+> 
+> It’s best to minimize the number of function calls to `=DCGETNAME(*dcids*)` by using a single call to get the names for a column of nodes. This is because tyhe spreadsheet will make one call to a Google server _[per custom function call](https://developers.google.com/apps-script/guides/sheets/functions#optimization)_. If your sheet contains many separate calls to `=DCGETNAME(*dcids*)`, you can expect it to be slow and return errors.
```

Comment:
```suggestion
> It’s best to minimize the number of function calls to `=DCGETNAME(*dcids*)` by using a single call to get the names for a column of nodes. This is because the spreadsheet will make one call to a Google server _[per custom function call](https://developers.google.com/apps-script/guides/sheets/functions#optimization)_. If your sheet contains many separate calls to `=DCGETNAME(*dcids*)`, you can expect it to be slow and return errors.
```